### PR TITLE
Fixed up time conversion code

### PR DESCRIPTION
**Pull request by ZeyadTamimi**

### CODE REVIEW

File: kaos/utils/time_conversion.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import datetime
+import calendar
+
+def utc_to_unix(time_string):
+    """Takes a string input of the format 'YYYYMMDDTHH:MM:SS.MS' and converts it to a UNIX
+    time stamp.
+
+    Args:
+        time_string (str): String representation of UTC time in the stated format. Must be
+                           greater 19700101.
```
Comment (lsav) [inline kaos/utils/time_conversion.py]: greater than

File: test/utils/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    @unpack
+    @data(('20181118T04:00:09.00', 1542513609),
+          ('19700101T00:00:00.00', 0),
+          ('19700101T00:00:12.00', 12),
+          ('19700101T01:06:40.00', 4000),
+          ('19700101T01:06:40.01', 4000.00),
+          ('19700101T01:06:40.40', 4000.00),
+          ('19700101T01:06:40.999', 4000.000),
+          ('23000912T01:01:01.00', 10435741261))
```
Comment (lsav) [inline test/utils/test_utils.py]: 👍

File: kaos/utils/time_conversion.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+"""This module contains all date/time related conversion code."""
+
+import datetime
+import calendar
+
+def utc_to_unix(time_string):
+    """Takes a string input of the format 'YYYYMMDDTHH:MM:SS.MS' and converts it to a UNIX
```
Comment (lsav) [inline kaos/utils/time_conversion.py]: Add a clarification here that the 'T' is the actual letter T

Comment (ZeyadTamimi) [reply kaos/utils/time_conversion.py]: Good point

**Comment by lsav** [comment]
No longer needs due to #17

**Pull request closed without merging**